x-pack/plugins/security_solution/public/resolver/view/index.tsx

```diff
@@ -123,17 +123,28 @@ export const Resolver = React.memo(function Resolver({
           tabIndex={0}
           aria-activedescendant={activeDescendantId || undefined}
         >
-          {edgeLineSegments.map(({ points: [startPosition, endPosition], metadata }, index) => (
-            <EdgeLine
-              edgeLineMetadata={metadata}
-              key={index}
-              startPosition={startPosition}
-              endPosition={endPosition}
-              projectionMatrix={projectionMatrix}
-            />
-          ))}
-          {[...processNodePositions].map(([processEvent, position], index) => {
-            const adjacentNodeMap = processToAdjacencyMap.get(processEvent);
+          {connectingEdgeLineSegments.map(
+            (
+              {
+                entity: {
+                  points: [startPosition, endPosition],
+                },
+              },
+              index
+            ) => (
+              <EdgeLine
```

Comment:
Looks like the `metadata` for the elapsed time was removed on the rebase probably

Reply:
thanks, did not intend to do that will add back.